### PR TITLE
NATIVE-622: Fix Sentry error reports

### PR DESCRIPTION
**Pull request by maxammann**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **NATIVE**.

### CODE REVIEW

File: build-configs/configs/BuildConfigType.js [hunk 1]
```diff
@@ -49,7 +49,8 @@ export type BuildConfigType = {|
   lightTheme: ThemeType,
   darkTheme: ThemeType,
   iconSet: string,
-  development?: boolean,
+  development: boolean,
+  sentry: boolean,
```
Comment (klinzo) [inline build-configs/configs/BuildConfigType.js:53]: Shuldn't this be a part of featureFlags?

**Review by klinzo — APPROVED** [review]
What was the problem? And how does this fix it?

File: build-configs/configs/BuildConfigType.js (hunk 1 above)
Comment (klinzo) [reply build-configs/configs/BuildConfigType.js:54]: Still relevant

Comment (maxammann) [reply build-configs/configs/BuildConfigType.js]: done

**Review by klinzo — APPROVED** [review]

**Pull request merged**